state/protocol/badger/mutator_test.go

```diff
@@ -1768,6 +1770,9 @@ func TestEpochFallbackMode(t *testing.T) {
 			require.NoError(t, err)
 			safetyThreshold := rootSnapshot.Params().EpochCommitSafetyThreshold()
 			require.NoError(t, err)
+			rootProtocolState, err := rootSnapshot.ProtocolState()
+			require.NoError(t, err)
+			epochExtensionViewCount := rootProtocolState.GetEpochExtensionViewCount()
```

Comment:
same here: would suggest to enforce that the returned value is larger or equal to `safetyThreshold`